components/slider/BrandSlider1.js

```diff
@@ -95,6 +95,18 @@ export default function BrandSlider1() {
             imageLight: '/images/exchanges/constellation.png',
             imageDark: '/images/exchanges/constellation-white.png',
             url: 'https://constellationnetwork.io/'
+        },
+        {
+            name: 'coinmarketcap',
+            imageLight: '/images/exchanges/cmc.png',
```

Comment:
The cmc.png and cg.png images don't actually appear to be black, they are also white like the others - not a big deal since we're using dark mode right now, but, it does break light mode in case we want to maintain that functionality. As it stands right now we could actually add a toggle for light mode and the whole site would still render and work - not sure if we want to continue on that path or not, but if we do, then we should probably take care to keep light mode functional with everything

Reply:
Updated and will push after the next review